DESCRIPTION

```diff
@@ -22,7 +22,9 @@ Imports:
   methods,
   rmarkdown (>= 1.10),
   rmdfiltr,
-  yaml
+  yaml,
+  lme4,
```

Comment:
I try to keep the mandatory dependencies of `papaja` to a minimum. To do so I add packages for which we provide `apa_print()`-methods to suggests. In this case, it would be necessary to additionally check if `broom.mixed` is installed, because we can't assume that `lme4` users necessarily also use that package. See [here](https://github.com/crsh/papaja/blob/92d5ad4b3740c3b5f8f742f41c7d2584aaf0a3a6/R/calculations.R#L9) for a solution to a similar issue with the `boot` package.